src/odb/include/odb/db.h

```diff
@@ -7476,30 +7549,99 @@ class dbModInst : public dbObject
 
   static dbModInst* getModInst(dbBlock* block_, uint dbid_);
 
-  std::string getName() const;
-
   std::string getHierarchicalName() const;
+
+  bool findModITerm(const char* name, dbModITerm*& ret);
+
+  dbSet<dbModITerm> getModITerms();
+
   // User Code End dbModInst
 };
 
+class dbModITerm : public dbObject
+{
+ public:
+  dbModInst* getParent() const;
+
+  void setNet(dbModNet* net);
+
+  dbModNet* getNet() const;
+
+  void setNextNetModiterm(dbModITerm* next_net_moditerm);
+
+  dbModITerm* getNextNetModiterm() const;
+
+  void setPrevNetModiterm(dbModITerm* prev_net_moditerm);
+
+  dbModITerm* getPrevNetModiterm() const;
+
+  void setNextEntry(dbModITerm* next_entry);
+
+  dbModITerm* getNextEntry() const;
+
```

Comment:
private

Reply:
automatically generated code. Keep as is.

class dbModBTerm : public dbObject
{
 public:
  const char* getName() const;

  dbModule* getParent() const;

  void setNet(dbModNet* net);

  dbModNet* getNet() const;

  void setNextNetModbterm(dbModBTerm* next_net_modbterm);